config/config.go

```diff
@@ -204,7 +204,7 @@ type Config struct {
 	RuleFiles      []string        `yaml:"rule_files,omitempty"`
 	ScrapeConfigs  []*ScrapeConfig `yaml:"scrape_configs,omitempty"`
 
-	RemoteWriteConfig RemoteWriteConfig `yaml:"remote_write,omitempty"`
```

Comment:
Mh, can we use this opportunity to structure this sanely into `remote.write.*`. There will be a read path that wants to be configured at some point.

Why adding `_config`. I know we have it everywhere else but it's basically completely pointless – everyone knows its a config because the whole file is a config.